### PR TITLE
1.7.0 release notes for Services Toolkit

### PR DESCRIPTION
**Pull request by gcapizzi**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

1.7.0 only.

### CODE REVIEW

File: release-notes.hbs.md [hunk 1]
```diff
@@ -151,6 +162,17 @@ This release includes the following changes, listed by component and area.
   about your external secrets and secret stores. The CREATE command lets you create cluster
   external secret and cluster secret stores. For more information, see the [Tanzu CLI Command Reference](https://docs.vmware.com/en/VMware-Tanzu-CLI/1.0/tanzu-cli/command-ref.html) documentation.
 
+#### <a id='1-7-0-service-bindings'></a> v1.7.0 Features: Service Bindings
+
+- Introduces the new `servicebinding.tanzu.vmware.com` package, which supersedes the existing
+  `service-bindings.labs.vmware.com`. The new package is based on the VMware-maintained
```
Comment (teddyking) [inline release-notes.hbs.md:168]: **issue** This is the other way round, the new Package is based on the community-maintained version

File: release-notes.hbs.md [hunk 2]
```diff
@@ -28,7 +28,8 @@ This release includes the following platform-wide enhancements.
   Tanzu Application Platform components are behaving as expected.
 
 - [AWS Services](aws-services/about.hbs.md): Provides a more streamlined approach for integrating
-  services from AWS into Tanzu Application Platform.
+  services from AWS into Tanzu Application Platform. Currently supports RDS PostgresSQL and MySQL. The package
+  is optional and depends on `crossplane.tanzu.vmware.com`.
```
Comment (teddyking) [inline release-notes.hbs.md:32]: **thought** it also depends on services-toolkit however I'm not sure we need to mention the dependency here as stk/crossplane are installed in all default profiles.

File: release-notes.hbs.md (hunk 1 above)
Comment (gcapizzi) [reply release-notes.hbs.md:168]: ah LOL, let me fix it immediately!

File: release-notes.hbs.md (hunk 2 above)
Comment (gcapizzi) [reply release-notes.hbs.md:32]: I wasn't sure either, but it was mentioned in the Google doc. Happy to remove it! Do we want to keep mentioning the optionality? I couldn't find any other similar reference in the file, so I'm not sure it's customary.

Comment (gcapizzi) [reply release-notes.hbs.md]: Done!

**Comment by teddyking** [comment]
LGTM!

File: release-notes.hbs.md
Comment (w110yd) [inline release-notes.hbs.md:1]: Nitpick: "on AWS" at the end just to make it very clear.

**Pull request merged**